server/api/account.js

```diff
@@ -0,0 +1,35 @@
+'use strict'
+
+const router = require('express').Router()
+const User = require('../db').models.user
+
+router.get('/', (req, res, next) => {
+  User.findById(req.session.user.id)
+  .then(user => {
+    res.status(200).json(user)
+  })
+  .catch(next)
+})
+
+router.put('/edit', (req, res, next) => {
+  User.update(req.body, 
+    {where: {id: req.session.user.id},
+    returning: true,
+    plain: true 
+  })
+  .then(user => {
+    res.status(200).json(user)
+  })
+  .catch(next)
+})
+
+router.delete('/delete', (req, res, next) => {
+  User.findById(req.session.user.id)
```

Comment:
maybe doesnt matter right now, but would we need to make sure to save this user's order data before deleting user